backend/api/quivr_api/modules/knowledge/service/knowledge_service.py

```diff
@@ -59,29 +73,30 @@ async def add_knowledge(
         return inserted_knowledge
 
     async def get_all_knowledge(self, brain_id: UUID) -> List[Knowledge]:
-        knowledges_models = await self.repository.get_all_knowledge_in_brain(brain_id)
+        all_knowledges_brain = await self.repository.get_all_knowledge_in_brain(
+            brain_id
+        )
 
         knowledges = [
             Knowledge(
-                id=knowledge.id,  # type: ignore
-                brain_id=knowledge.brain_id,
-                file_name=knowledge.file_name,
-                url=knowledge.url,
-                mime_type=knowledge.mime_type,
-                status=KnowledgeStatus(knowledge.status),
-                source=knowledge.source,
-                source_link=knowledge.source_link,
-                file_size=knowledge.file_size
-                if knowledge.file_size
+                id=knowledge_brain.knowledge.id,  # type: ignore
```

Comment:
Not a necessary change: Pydantic should have a better interface for creating a model from another. This would cleanup this part